addonscript/src/addonscript/xpi.py

```diff
@@ -1,10 +1,29 @@
 """Resources that operate on an XPI file."""
 
 import json
+import re
 from zipfile import ZipFile
 
+from addonscript.exceptions import BadVersionError
 
-def get_langpack_info(context, path):
+
+def get_stripped_version(version):
```

Comment:
@JohanLorenzo Does MozillaVersion have a better solution here? (Even if so, this could be a followup).